### ISSUE TITLE
Abbreviations ABC -> Abc

### ISSUE THREAD

**Issue by H4rryK4ne** [wartaal/HanTa#8]
I have a text with a lot of abbreviations (VT, KW od KW47, MWG, EKV, ...) and most (if not all) of them are tagged as NE and lemmatized as (Vt, Kw oder Kw47, Mwg, Ekv, ...).

**Comment by wartaal** [comment]
Yes, I know. This is not so easy to solve unless I would write 100 Heurisic Rules, since there are MANY POSSIBILITIES how capitalization can be used. If you know what kind of text you are dealing with some simple post-processing would be the best option, I think.
